include/engine/data_watchdog.hpp

```diff
@@ -82,7 +83,7 @@ class DataWatchdog
     std::thread watcher;
```

Comment:
One step further is to make `storage::SharedDataTimestamp` parameterized with `AlgorithmT`.
It will give possibility later update only one specific data portion in `/dev/shm/osrm-region-ch`,  `/dev/shm/osrm-region-core-ch` or ` `/dev/shm/osrm-region-mld``